keyboards/kprepublic/bm40hsrgb/keymaps/wolff_abnt2/rules.mk

```diff
@@ -0,0 +1,25 @@
+MCU = atmega32u4
+BOOTLOADER = atmel-dfu
+EXTRAFLAGS += -flto
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+TERMINAL_ENABLE = no
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+AUDIO_ENABLE = no           # Audio output
+SPLIT_KEYBOARD = no
+KEY_LOCK_ENABLE = no
+RGB_MATRIX_ENABLE = yes
+RGB_MATRIX_DRIVER = WS2812
+LAYOUTS = planck_mit
+LAYOUTS_HAS_RGB = yes
```

Comment:
Not going to check everything, but a lot of these are already done in the keyboard-level's `rules.mk` too

Reply:
Deleted most dupes; kept a few ones (the most likely to be modified at a keymap level)